docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md

```diff
@@ -0,0 +1,211 @@
+
+Using the above example configuration, specify the connection string for your database.  Then run this code to test your
+configuration:
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L75
+```
+
+You will then see your tables listed as `Available data_asset_names` in the output of `test_yaml_config()`.
+
+Feel free to adjust your configuration and re-run `test_yaml_config()` as needed.
+
+### 4. Save the Datasource configuration to your DataContext
+
+Save the configuration into your `DataContext` by using the `add_datasource()` function.
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L77
+```
+
+## Introspection
+
+### 1. Get names of available Data Assets
+ 
+Getting names of available data assets using the data connector keys referenced in the `introspection` section of the
+configuration gives you visibility into the tables in your database:
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L78-L80
+```
+
+### 2.  Examine a few rows of a table
+
+Pick a `data_asset_name` from the previous step and specify it in the `BatchRequest`:
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L84-L88
+```
+
+Then load data into the `Validator` and print a few rows of the table (`n_rows = 5` is the default, but you may look at
+more rows e.g. `validator.head(n_rows=100)`):
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L94-L99
+```
+
+At this point, you can also perform additional checks, such as confirming the number of batches and the size of a batch.
+For example (be sure to adjust this code to match the specifics of your data and configuration):
+
+```python file=../../../tests/integration/docusaurus/connecting_to_your_data/how_to_introspect_and_partition_your_data/sql_database/yaml_example.py#L102-L108
+```
+
+### 3. Using Splitting to aid introspection
+
+`Splitting` is a `Data Connector` capability that provides the means of breaking the batch data up based on the values
+of certain dimensions of the data of interest.  Employing `Splitting` as part of introspection can help focus on smaller
+batches, centered around a common aspect of the data.
+
+To configure `Splitting`, specify a dimension (i.e., `column_name` or `column_names`), the method of splitting, and
+parameters to be used by the specified splitting method.
+
+For the present example, we can split according to the "pickup_datetime" column parsed to the date level precision and
+thus obtain batches containing the taxi rides data for each day found in the available data:
```

Comment:
Wanting to check formatting. Should "pickup_datetime" be `pickup_datetime`?

Reply:
Fixed!  Nice catch @Shinnnyshinshin !  Thanks!